code/game/objects/items/devices/taperecorder.dm

```diff
@@ -22,6 +22,12 @@
 	var/canprint = TRUE
 	var/list/icons_available = list()
 	var/icon_directory = 'icons/effects/icons.dmi'
+	///Whether we've warned during this recording that the tape is almost up.
+	var/seconds_warned = FALSE
```

Comment:
This var name sounds more like the seconds since you warned someone, is that intended or just my brain playing tricks

Reply:
Yeah it's hard to understand because of brevity I'll try to think of a better name